### PR TITLE
Change Pspamm installation documentation

### PR DESCRIPTION
**Pull request by NicoSchlw**
Correct the PSpaMM symlink in the docs

### CODE REVIEW

**Comment by ravil-mobile** [comment]
Hi, could you also mention that `pspamm` can be installed with Spack
```
spack install py-pspamm
```
it will require Spack version >=0.17.2

**Comment by Thomas-Ulrich** [comment]
This won't work if bin does not exist.
What you need is mkdir ~/bin before the symbolic link.

**Comment by ravil-mobile** [comment]
In general, one needs to just clone pspamm to a directory and then append `PATH` and `PYTHONPATH` with the name of that directory

**Comment by NicoSchlw** [comment]
So should we just provide the lines for the bashrc in the docs?

**Comment by NicoSchlw** [comment]
Is appending to PYTHONPATH really necessary?

**Comment by krenzland** [comment]
Nope, never done that.

File: Documentation/compilation.rst [hunk 1]
```diff
@@ -99,7 +99,14 @@ Installing PSpaMM
...
    git clone https://github.com/SeisSol/PSpaMM.git
-   ln -s $(pwd)/PSpaMM/pspamm.py $HOME/bin
+   # make sure $HOME/bin exists or create it with "mkdir ~/bin"
+   ln -s $(pwd)/PSpaMM/pspamm.py $HOME/bin/pspamm.py
+   
+Instead of linking, you could also add the following line to your .bashrc:
+
+.. code-block:: bash
+
+   export PATH=<Your_Path_to_PSpaMM>/PSpaMM:$PATH
```
Comment (krenzland) [inline Documentation/compilation.rst:109]: This line is a bit confusing, shouldn't it be
export PATH=<Your_Path_to_PSpaMM>:$PATH
?

**Review by krenzland — CHANGES_REQUESTED** [review]

**Review by krenzland — APPROVED** [review]

**Comment by krenzland** [comment]
Thanks!

**Pull request merged**